benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -691,5 +696,8 @@ msgstr ""
 "La página que estás buscando puede estar en otro lugar o puede que ya no "
 "exista."
 
+#, fuzzy
+#| msgid "core.icons.sadbus"
+msgctxt "image alt text"
 msgid "core.icons.sadbus"
```

Comment:
I don't know why this fuzzy is here!

Reply:
https://github.com/cal-itp/benefits/pull/1566#discussion_r1273644826